spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/spring-cloud-dataflow-features.adoc

```diff
@@ -0,0 +1,65 @@
+[[spring-cloud-data-flow-features]]
+== Features
+
+* Orchestrate applications across a variety of distributed modern runtimes including: Cloud Foundry, Apache YARN, Apache Mesos, and Kubernetes
```

Comment:
If all servers (including the "local" one) have the same paragraphs, why not make this 65lines file the thing we include?